### DISCUSSION TITLE
Getting versions right

### DISCUSSION THREAD

**Discussion by gideonsimpson** [jupyterlab/jupyterlab-git#924] [Q&A]
I've had some trouble of late getting this module to work properly with jupyterlab 3.x.  I've a conda installation of jupyterlab 3.0.11, and when I try to add it, conda installs jupyterlab-git-0.22.0, but it does not seem to be functional.  If I then click on the extensions tab of jupyterlab, it shows up with a red border, along with nbdime.  Any suggetsions?

**Comment by krassowski** [comment]
It is currently in beta due to the beta status of a dependency (which is being worked on). To get the latest beta release you can use:

```
pip install --pre jupyterlab-git==0.30.0b3
```

But it seems that there is no beta release on conda-forge indeed. Please be advised that mixing conda and pip may result in (difficult to diagnose) issues, so ideally once the final version lands on conda-forge, you should first uninstall the beta version installed with pip and only then install the stable version using conda.

**Reply by krassowski** [reply]
Also, note that neither `jlpm install` nor `jlpm build` steps are required for JupyterLab 3.0+, and it even appears that using those may lead to issues. I recommend only using pip/conda way of installing extension for JupyterLab 3 (if possible).
